.gitignore

```diff
@@ -102,3 +102,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# Augur
+*.aug
```

Comment:
What is a ```.aug``` file?

Reply:
`.aug` is the extension I used for the text files for passing in the augur type definitions from the documentation. I accidentally committed my personal `.gitignore` file it seems.

Reply:
Okay no problem, remove this then from the repo.